redaxo/src/core/fragments/core/header.php

```diff
@@ -9,9 +9,9 @@
             <div class="container-fluid">
                 <div class="navbar-header">
                     <?php if (rex_be_controller::getCurrentPageObject()->isPopup()): ?>
-                        <span class="navbar-brand"><img class="rex-js-svg rex-redaxo-logo" src="<?= rex_url::coreAssets('redaxo-logo.svg') ?>" /></span>
+                        <span class="navbar-brand"><?= rex_file::get(rex_url::coreAssets('redaxo-logo.svg')) ?></span>
```

Comment:
```suggestion
                        <span class="navbar-brand"><?= rex_file::get(rex_path::coreAssets('redaxo-logo.svg')) ?></span>
```

rex_path liefert den absoluten Pfad, rex_url hingegen den relativen für URLs. Was durchaus auch klappt bei Dateioperationen, aber absoluter Pfad ist da besser.

Reply:
entsprechend die anderen stellen auch noch.